pkg/orchestrator/etcd_worker.go

```diff
@@ -177,6 +189,25 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 }
 
 func (worker *EtcdWorker) handleEvent(_ context.Context, event *clientv3.Event) {
+	if string(event.Kv.Key) == worker.prefix.String()+deletionCounterKey {
+		switch event.Type {
+		case mvccpb.PUT:
+			var err error
+			worker.deleteCounter, err = strconv.ParseInt(string(event.Kv.Value), 10, 64)
+			if worker.deleteCounter <= 0 {
+				log.Panic("unexpected delete counter", zap.Int64("value", worker.deleteCounter))
```

Comment:
Could you log key and value?

Reply:
The key is actually fixed. There is only one possible value if this occurs in running the cdc server.